custom-transforms-example/config/topology.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 sources:
-  - Redis:
+  - Valkey:
       name: "redis"
```

Comment:
Do we want to rename this opportunistically as well? Or it should be left to the following PRs?

Reply:
I will do them in follow up PRs